charts/spire/templates/spire-controller-manager-config.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.controllerManager.enabled }}
```

Comment:
can we rename the file to spire-conroller-manager-configmap.yaml to keep same naming convention?